### PR TITLE
Update viva-learning-activity.md

### PR DESCRIPTION
**Pull request by msbemba**
Added the type of user activities that are available in the report

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10661

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/activity-reports/viva-learning-activity.md [hunk 1]
```diff
@@ -40,5 +40,8 @@ You can get a view into your user's Viva Learning activity by looking at the **A
 ![Microsoft 365 Apps usage report.](../../media/viva-learning-charts.png)
 
 - Active users – Shows you the number of active users over time. For example, you can use it to find the number of active users in your organization on a specific day over the past 30 days.
-- User activities – Shows you the number of user activities over all platforms. For example, you can use it to find how many users in your organization bookmarked a course in the last 30 days.
+- User activities – Shows you the number of user activities such as Viewed Featured Content, LearningTabsPinned, Viewed Learning Objects, Consumed Learning Objects, Recommendations made, and Bookmarks.
+
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/viva-learning-activity.md]:
```suggestion
- User activities – Shows you the number of user activities, such as viewed featured content, learning tabs pinned, viewed learning objects, consumed learning objects, recommendations made, and bookmarks.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@camillepack Please merge this PR. Thanks

**Pull request merged**